packages/tools/lib/generate-json-imports/illustrations.js

```diff
@@ -0,0 +1,75 @@
+const fs = require("fs").promises;
+const path = require('path');
+
+const generate = async () => {
+	const fioriInputFolder = path.normalize(process.argv[2]);
+	const tntInputFolder = path.normalize(process.argv[3]);
+	const outputFile = path.normalize(`${process.argv[4]}/Illustrations.js`);
+
+	const dirs = await fs.readdir(fioriInputFolder);
+	const fioriIllustrationsOnFileSystem = dirs.map(illustrationName => {
```

Comment:
Here AllIllustrutaion.js is also matched, you can see below that it leads to creation of a loader for it.